File: HLC01_02Java/src/main/java/org/example/Menu.java
package org.example;

import java.util.*;


public class Menu {
    private Actividades actividades = new Actividades();
    private Scanner scanner = new Scanner(System.in);


    public String askForOption() {
        System.out.print("\nIntroduce el numero de la actividad (1-14) o 0 para salir: ");
        String option = scanner.nextLine();
        System.out.print("\n");
        return option;
    }


    public void performOption(String option) {
        switch (option) {
            case "0":
                System.out.println("Adios");
                break;
            case "1":
                actividades.ej01();
                break;
            case "2":
                actividades.ej02();
                break;
            case "3":
                actividades.ej03();
                break;
            case "4":
                actividades.ej04();
                break;
            case "5":
                actividades.ej05();
                break;
            case "6":
                actividades.ej06();
                break;
            case "7":
                actividades.ej07();
                break;
            case "8":
                actividades.ej08();
                break;
            case "9":
                actividades.ej09();
                break;
            case "10":
                actividades.ej10();
                break;
            case "11":
                actividades.ej11();
                break;
            case "12":
                actividades.ej12();
                break;
            case "13":
                actividades.ej13();
                break;
            case "14":
                actividades.ej14();
                break;
            default:
                System.out.println("La opción introducida no existe...");
                break;
        }
    }
}

File: HLC01_02Kotlin/src/main/kotlin/Menu.kt
class Menu {

    fun askForOption(): String {
        print("\nIntroduce el numero de la actividad (1-14) o 0 para salir: ")
        val option = readln()
        print("\n")
        return option
    }

    fun performOption(option: String) {
        when (option) {
            "0" -> println("Adios")
            "1" -> Actividades.ej01()
            "2" -> Actividades.ej02()
            "3" -> Actividades.ej03()
            "4" -> Actividades.ej04()
            "5" -> Actividades.ej05()
            "6" -> Actividades.ej06()
            "7" -> Actividades.ej07()
            "8" -> Actividades.ej08()
            "9" -> Actividades.ej09()
            "10" -> Actividades.ej10()
            "11" -> Actividades.ej11()
            "12" -> Actividades.ej12()
            "13" -> Actividades.ej13()
            "14" -> Actividades.ej14()
            else -> println("La opción introducida no existe...")
        }
    }

}